src/inverseRobot.js

```diff
@@ -7,7 +7,18 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const clone = {};
```

Comment:
The variable name 'clone' could be more descriptive. Consider renaming it to 'invertedRobot' or something similar to reflect its purpose of storing the inverted key-value pairs of the 'robot' object.